boards/nordic/nrf54l20pdk/board.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
+
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
```

Comment:
nrfjprog is not supported for this soc: cc: @carlescufi

Reply:
Yep, please use only nrfutil @kl-cruz

Reply:
done